pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -48,6 +50,9 @@ func NewConfigObserver(
 		builds.ObserveBuildControllerConfig,
 		network.ObserveExternalIPAutoAssignCIDRs,
 		deployimages.ObserveControllerManagerImagesConfig,
+		//TODO when https://github.com/openshift/api/pull/982 merges we can add "CSIDriverSharedResource" to that
+		// currently includes "BuildCSIVolumes"
```

Comment:
@adambkaplan - conceivably, we don't need both CSIDriverSharedResource that you cited in BUILD-292 and BuildCSIVolumes that you cited in BUILD-281, unless you think CSI shared resource stuff could move out of tech preview mode in a release prior to when Build's consumption of it move out of tech preview mode.

Reply:
UPDATE: @adambkaplan and I talked and he reminded me that long term we will want to support multiple CSI volume types for builds

hence the multiple feature gates do make sense.